sdks/java/maven-archetypes/starter/pom.xml

```diff
@@ -60,6 +60,9 @@
               <goals>
                 <goal>integration-test</goal>
               </goals>
+              <configuration>
+                <ignoreEOLStyle>true</ignoreEOLStyle> <!-- for win -->
```

Comment:
Can you remove the comment? Alternatively, please expand as to why it is needed, etc.